docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   db:
-    image: postgres:10-alpine
```

Comment:
We should just update the version in this PR.

Reply:
Ups, yes :)

Reply:
Fixed, removed the commit upload by mistake.